bus/pom.xml

```diff
@@ -49,40 +49,37 @@
     </dependency>
 
     <dependency>
-      <groupId>org.hawkular.bus</groupId>
-      <artifactId>hawkular-bus-common</artifactId>
-      <version>0.0.5</version>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging</artifactId>
     </dependency>
 
-    <!-- these will be provided by our RA - the MDB itself will never need ActiveMQ specific classes -->
     <dependency>
-      <groupId>org.apache.activemq</groupId>
-      <artifactId>activemq-all</artifactId>
-      <version>${version.org.apache.activemq}</version>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging-annotations</artifactId>
       <scope>provided</scope>
     </dependency>
 
     <dependency>
-      <groupId>org.apache.activemq</groupId>
-      <artifactId>activemq-jaas</artifactId>
-      <version>${version.org.apache.activemq}</version>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging-processor</artifactId>
       <scope>provided</scope>
     </dependency>
 
+    <!-- Test -->
     <dependency>
-      <groupId>org.jboss.logging</groupId>
-      <artifactId>jboss-logging</artifactId>
+      <groupId>org.hawkular.bus</groupId>
+      <artifactId>hawkular-bus-test-common</artifactId>
+      <version>0.3.0.Final</version>
```

Comment:
You should not be hardcoding component versions like this.  Extract out into a <version.org.hawkular.bus> property at least in the top parent pom.xml of your component (we should probably put these in parent pom in the future)